src/components/NotFoundPage.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
```

Comment:
I don't think you need to import React in vite projects if you are not actually using the variable, but not a big deal.